src/styles/card.scss

```
@@ -0,0 +1,89 @@
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  font-size: 12px;
+}
+
+.card {
+  width: 200px;
+  height: 408px;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__product-picture {
+    width: 166px;
+    height: 134px;
+    background-image: url(../images/imac.jpeg);
+    background-size: 160px 134px;
+    background-repeat: no-repeat;
+    background-position: center;
+    margin-top: 32px;
+    margin-bottom: 40px;
+    margin-inline: $margin-inline;
```

since we'll apply flex and padding to a card
```suggestion
    margin: 0 auto 40px;
```

---

ok